lib/msf/core/auxiliary/cisco.rb

```diff
@@ -280,6 +280,20 @@ def cisco_ios_config_eater(thost, tport, config)
             create_credential_and_login(cred)
           end
 
+        # This regex captures ephones from Cisco Unified Communications Manager Express (CUE) which come in forms like:
+        # username &#34;phonefour&#34; password 444444
+        # username test password test
+        # This is used for the voicemail system
+        when /^\s*username (?:&#34;)([^\s]+)(?:&#34;) password ([^\s]+)/i
+          user  = $1
+          spass = $2
+          print_good("#{thost}:#{tport} ePhone Username '#{user}' with Password: #{spass}")
+          store_loot("cisco.ios.ephone.username_password", "text/plain", thost, "#{user}:#{spass}", "ephone_username_password.txt", "Cisco IOS ephone Username and Password")
+            cred = credential_data.dup
```

Comment:
Inconsistent indentation

Reply:
why my inner python dev didn't scream about this, i'm not sure, i blame copy pasta.